packages/lascar/lascar.0.6/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+synopsis: "A library for manipulating Labeled Transition Systems in OCaml"
+maintainer: ["Jocelyn Sérot <jocelyn.serot@uca.fr>"]
+authors: ["Jocelyn Sérot <jocelyn.serot@uca.fr>"]
+license: "MIT"
+homepage: "https://github.com/jserot/lascar"
+bug-reports: "jocelyn.serot@uca.fr"
+depends: [
+  "dune" {build & >= "1.11"}
```

Comment:
```suggestion
  "dune" {>= "1.11"}
```
Dune should not be tagged as a build dependency (see #14266)

Reply:
Ok